contracts/OffsetHelper.sol

```diff
@@ -449,84 +492,40 @@ contract OffsetHelper is OffsetHelperStorage {
         );
 
         // approve router
-        IERC20(_fromToken).safeApprove(sushiRouterAddress, _fromAmount);
+        IERC20(_fromToken).safeApprove(dexRouterAddress, _fromAmount);
 
         // swap
-        uint256[] memory amounts = routerSushi().swapExactTokensForTokens(
+        uint256[] memory amounts = dexRouter().swapExactTokensForTokens(
             _fromAmount,
             0, // min. output amount
             path,
             address(this),
             block.timestamp
         );
-        uint256 amountOut = amounts[len - 1];
+        amountOut = amounts[len - 1];
 
         // update balances
-        balances[msg.sender][_toToken] += amountOut;
-
-        return amountOut;
-    }
-
-    // apparently I need a fallback and a receive method to fix the situation where transfering dust MATIC
-    // in the MATIC to token swap fails
-    fallback() external payable {}
-
-    receive() external payable {}
-
-    /**
-     * @notice Return how much MATIC is required in order to swap for the
-     * desired amount of a Toucan pool token, for example, BCT or NCT.
-     *
-     * @param _toToken The address of the pool token to swap for, for
-     * example, NCT or BCT
-     * @param _toAmount The desired amount of pool token to receive
-     * @return amounts The amount of MATIC required in order to swap for
-     * the specified amount of the pool token
-     */
-    function calculateNeededETHAmount(address _toToken, uint256 _toAmount)
```

Comment:
Why are these  functions being removed?

Reply:
not removed. I just re-organized the functions because there was no structure in the file for public, internal and private functions. 


only happened in the second last commit.